build.gradle.kts

```diff
@@ -11,7 +11,6 @@ description = "SAML Conformance Test Kit"
 
 buildscript {
     repositories {
-        jcenter()
```

Comment:
He just moved it down to the `allProjects` closure instead of this one.
https://github.com/codice/saml-conformance/pull/208/files/3cdfc4fae044b5f939a368abfae1eb4f1ea0f310#diff-392475fdf2bc320d17762ed97109a121R39